linkedList.js

```diff
@@ -4,5 +4,110 @@
  * @return {Object} an object exposing methods to be used to manipulate a linked list
  */
 function linkedListGenerator(){
+  var head = null;
+  var tail = null;
+  var node = function() {
```

Comment:
Usually we use capitalized function names for constructors as convention. I would refactor this to be `Node`.